index.html

```diff
@@ -10,22 +10,19 @@
 </head>
 <body>
     <div id="headline">
-        <header id="headerr">
-            <div id="nava">
-                <a class="myLogo" href="#">Abedy</a>
-                <nav>
-                    <img id="navv" src="icons/Union.png" alt="Union.png">
-                    <img class="closebtn" src="icons/Icon.png" alt="x imge">
-                </nav>
-            </div>
-        </header>  
-        <div class="menu">
-            <ul class="menu-items">
-                <li><a class="menu-item" href="#cards">Portfolio</a></li>
-                <li><a class="menu-item" href="#about-me">About</a></li>
-                <li><a class="menu-item" href="#contactform">Contact</a></li>
+        <header>
+            <span ><a class="myLogo" href="#">Abedy</a></span>
+            <nav>
+                <a href="#">
+                    <img src="icons/Union.png" alt="Union.png">
+                </a>
+            </nav>
+            <ul class="navlinks">
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
             </ul>
```

Comment:
- [ ] Kindly take a second look at this link so it would fit the Figma, area to check is the gap between the ul.

### Your design
![image](https://github.com/AbednegoKibitok/module1microverse/assets/86200356/25c7b4de-c170-453d-ae4f-c257f40142f7)

### Figma
![image](https://github.com/AbednegoKibitok/module1microverse/assets/86200356/ff4c0a7d-ef3b-4ea9-bfd4-b35b42604d35)